htdocs/api/v0.0.2/candidates/Visit.php

```diff
@@ -196,7 +196,12 @@ public function handlePUT()
                 $this->safeExit(0);
             }
             // need to extract subprojectID
-            $this->createNew($this->CandID, $subprojectID, $this->VisitLabel);
+            $this->createNew(
+                $this->CandID,
```

Comment:
Might be nice to use the CandID class here. Unless you want to wait for #4350

Reply:
This was just the minimal change to update the API call. It should be passing the \Candidate class itself and match the static function that it's delegating to, but this is dead code once #4244 gets merged anyways.